twingly_search/client.py

```diff
@@ -52,34 +62,41 @@ def query(self):
         """
         return Query(self)
 
-    def execute_query(self, query):
+    def execute_query(self, q):
```

Comment:
There is no method overloading in Python :-(
I'd prefer having 2 methods, where q parameter is either Query object or String.

What do you think about following syntax:
```python
def execute_query(self, q=None, query_string='')
```
Where one of the arguments should be supplied. If both are supplied, we should prefer query_string.